datafusion/src/logical_plan/builder.rs

```diff
@@ -220,10 +223,33 @@ impl LogicalPlanBuilder {
         for e in expr {
```

Comment:
I wonder if you could call `expand_wildcard` directly from here rather than duplicating the logic.

Reply:
refactored the code to use same expand_wildcard implementation in both places.